tests/shared/src/main/scala/munit/HedgehogFrameworkSuite.scala

```diff
@@ -42,8 +42,7 @@ class HedgehogFrameworkSuite extends HedgehogSuite {
   }
 
   property(
-    "custom config",
-    config = propertyConfig.copy(testLimit = SuccessCount(1000))
+    "custom config".tag(HedgehogConfig(hedgehogPropertyConfig.copy(testLimit = SuccessCount(1000))))
```

Comment:
This is quite a mouthful, I wonder if we want to add custom tags for the most common configuration options? Just brainstorming, it would be nice if the user could do something like this instead 🤔 

```suggestion
    "custom config".tag(TestLimit(1000))
```

Reply:
If we go down this route I would still namespace the tags

```suggestion
    "custom config".tag(HedgehogTestLimit(1000))
```

Reply:
This is totally optional BTW, just a suggestion. I'm happy to merge this PR as is. I believe we can update this PR to skip projects where the cross-build is missing

Reply:
I agree it's too verbose as-is. I'll update like you suggested. I think I'll add support for passing a seed as a tag as well.

I can also remove the `propertyF` stuff for now (until Hedgehog merges support for that).

One note about tags - figuring out the supported tags is not discoverable via auto-completion. We could address that via extension methods - what do you think?